### PR TITLE
Feat: only flag

### PR DESCRIPTION
**Pull request by Fabian-Fynn**
Only start specified scripts.

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 yarn exec lint-staged
+npm run lint
```
Comment (kaufmo) [inline .husky/pre-commit:5]: use yarn here

File: src/commands/start.command.ts [hunk 2]
```diff
@@ -6,12 +6,26 @@ import { createServer, Socket } from "net";
...
     let shuttingDown = false;
+    let scriptsToStart: ScriptDefinition[] = [];
+
+    if (options?.only && options?.only?.length > 0) {
+        options.only.map((scriptName) => {
+            const foundScript = scripts.find((script) => script.name === scriptName);
+            if (foundScript) {
+                scriptsToStart.push(foundScript);
+            } else {
+                console.log(`${colors.bgRed.bold.black(" dev-pm ")} Script ${scriptName} not found in dev-pm config`);
```
Comment (johnnyomair) [inline src/commands/start.command.ts:23]: We probably should use `console.error` here. In my opinion, we might even abort execution completely.

File: .husky/pre-commit (hunk 1 above)
Comment (nsams) [reply .husky/pre-commit:5]: why lint-staged *and* run lint?

**Review by nsams — APPROVED** [review]

File: .husky/pre-commit (hunk 1 above)
Comment (Fabian-Fynn) [reply .husky/pre-commit:5]: Yeah `yarn exec lint-staged` should be enough

File: src/commands/start.command.ts (hunk 2 above)
Comment (Fabian-Fynn) [reply src/commands/start.command.ts:23]: Good idea
I switched to `console.error` and abort execution using `process.exit(1);`

**Review by johnnyomair — APPROVED** [review]

**Comment by johnnyomair** [comment]
@Fabian-Fynn we should consider adding a changelog for the Dev PM

**Comment by Fabian-Fynn** [comment]
> @Fabian-Fynn we should consider adding a changelog for the Dev PM

Sounds good. Should it be starting from this point or retroactive?

**Comment by johnnyomair** [comment]
> Sounds good. Should it be starting from this point or retroactive?

As the Dev PM is relatively new, we should do it retroactively. It can be very short for now:

```
# [Next]

## Highlights

- Add `--only` flag to start specific microservices

# 1.0.0

Initial release

**Comment by Fabian-Fynn** [comment]
Ok I've added the changelog you suggested

**Comment by dkarnutsch** [comment]
@kaufmo please review

**Review by kaufmo — APPROVED** [review]

**Pull request merged**